bin/metrics-elb.rb

```diff
@@ -52,7 +52,7 @@ class ELBMetrics < Sensu::Plugin::Metric::CLI::Graphite
          description: 'Metric naming scheme, text to prepend to metric',
          short: '-s SCHEME',
          long: '--scheme SCHEME',
-         default: ''
+         default: 'loadbalancer'
```

Comment:
this technically is also a breaking change and should be called out as well. I'd also suggest that maybe elb names more sense as there are now the following types of load balaners in aws and where possible I'd go for more specific:
- ElasticLoadBalancer
- NetworkLoadBalancer
- ApplicationLoadBalaner

Reply:
as the name implies I'd prefer `elb`

Reply:
I'm not sure to follow what you are asking for here: would you prefer to have ``elb`` or one of the other more specific name? I guess we could have this scheme auto configured respectively to the actual load balancer being monitored, but in that case I would make that into another PR.

As for the value itself:
* it's a breaking change compared to the ``-full`` variant, I'll document it in the changelog.
* I will change it to ``elb`` if you prefer. That would be a breaking change for the normal variant as well in this case (which had ``loadbalancer`` hardcoded.)